file-formats/srvb/type/zif_aff_srvb_v1.intf.abap

```diff
@@ -0,0 +1,80 @@
+INTERFACE zif_aff_srvb_v1
+  PUBLIC.
+  TYPES:
+    "! <p class="shorttext">Service Content</p>
+    "! Service content
+    BEGIN OF ty_service_content,
+      "! <p class="shorttext">Service Major Version</p>
+      "! Service Major Version
+      "! $required
+      service_version       TYPE c LENGTH 4,
+      "! <p class="shorttext">Service Minor Version</p>
+      "! Service Minor Version
+      "! $required
+      service_minor_version TYPE n LENGTH 10,
+      "! <p class="shorttext">Service Patch Version</p>
+      "! Service Patch Version
+      "! $required
+      service_patch_version TYPE n LENGTH 10,
+      "! <p class="shorttext">Service Build Version</p>
+      "! Service Build Version
+      service_build_version TYPE n LENGTH 10,
+      "! <p class="shorttext">API Release State</p>
+      "! API Release State
+      "! $required
+      release_state         TYPE string,
+      "! <p class="shorttext">Service Definition</p>
+      "! Service Definition
+      "! $required
+      srvd_name             TYPE string,
```

Comment:
could this be type c length 30. Abap objects usually only consist of a maximum of 30 characters

Reply:
Taken care in the latest commit